changelog/15525.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+auth/ldap: A bug during login has been fixed when `username_as_alias` is set.
```

Comment:
Nit: Can we update this entry to be a little bit more descriptive on the nature of the bug? Maybe also include the keyword `userattr` somewhere in here to to make it a bit more discoverable/searchable.